### ISSUE TITLE
Support for named columns

### ISSUE THREAD

**Issue by aylabyuk** [Mando75/typeorm-graphql-loader#13]
named columns are not supported by the package. so for example this entity:

```
@ObjectType()
@Entity()
export class Author extends BaseEntity {
  ...
  @Field()
  @Column("varchar", { name: "mobile" })
  phone!: string;
...
}
```

the column "phone" has its original name as "mobile" in the database.
right now when you query against this model all the values for the other columns are returned but not for this column.

**Possible Solution:** 
I was able to fix this by adding/changing some lines in the GraphQLQueryResolver.ts file.
specifically on line 86 - 95. 

fields.forEach(field => {
      // Make sure we account for embedded types
      const propertyName: string = field.propertyName;
      const databaseName: string = field.databaseName;

      queryBuilder = queryBuilder.addSelect(
        this._formatter.columnSelection(alias, propertyName),
        this._formatter.aliasField(alias, databaseName)
      );
    });

I used the field.databaseName as the alias to access the value for each field.

I will try add a PR, as soon as I was able get the tests ready.

**Comment by Mando75** [comment]
Thanks for the PR. I'll take a look at it tonight and run it through some more tests. Assuming everything works correctly I'll get your PR merged and a patch published tonight. 

EDIT: Sorry didn't mean to close the issue. Fat fingers :/

**Comment by Mando75** [comment]
Just one other note here, as mentioned in the CONTRIBUTING section of the README, this GitHub repo is actually just a mirror of a GitLab repo where the development actually takes place. As such, I can't actually merge the PR in GitHub as all of the deployment/publishing pipelines won't get run in GitLab. If you have a GitLab account, I encourage you to open an MR for this repository https://gitlab.com/Mando75/typeorm-graphql-loader. Otherwise, I can just pull in your changes as a feature branch and handle merging it that way.

**Comment by Mando75** [comment]
Took a look through everything and it seems to work great. I did have to make a couple tweaks to get it to work correctly with embedded entities as well, but I'm shipping the fix here in a minute under version 1.1.1 of the package. Like I mentioned, I actually ship the package through GitLab, so I will close this issue and your PR. Rest assured, you fix made it in though. Thanks again for bringing this up. :)

**Comment by aylabyuk** [comment]
Cheers @Mando75! really awesome package by the way, this fits most of my use case in my current project :+1: I will try to contribute more.. of course through gitlab.
